lib/views/movie_list_view.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+import '../custom_widgets/movie_info.dart';
+import '../repositories/movie_repository.dart';
+import '../entities/movie.dart';
+
+class MovieListView extends StatefulWidget {
+  const MovieListView({super.key});
+
+  @override
+  State<MovieListView> createState() => _MovieListViewState();
+}
+
+class _MovieListViewState extends State<MovieListView> {
+  final MovieRepository movieRepository = MovieRepository();
+  Future<List<Movie>>? movies;
```

Comment:
This should be `Future<List<Movie>> movies`, as that's the return type of the repository method

Reply:
done